lib/actions/queue/GetQueueMetadata.js

```diff
@@ -10,11 +10,12 @@ class GetQueueMetadata {
     process(request, res) {
         const queue = QueueManager.getQueueAndMessage({ queueName: request.queueName }).queue,
             metaProps = queue.metaProps,
+            queueLength = queue.getLength(),
             response = new AzuriteQueueResponse();
-        response.addMetaProps(metaProps);
+        response.addMetaProps(metaProps, queueLength);
```

Comment:
@leafnode can you please use response.addHttpProperty('x-ms-approximate-messages-count', queueLength);

Reply:
Sure! Actually, adding more parameters to the signature looked bad, but I wanted to keep this in one place and didn't have much time for deeper refactoring. But if adding it in another line is fine - I'm good with it.